markdown-editor/src/Components/App.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
```

Comment:
You can access hooks and other react methods on the sample import line

```suggestion
import React, {useState, createContext} from "react";
```